src/builder/edit_channel.rs

```diff
@@ -311,21 +302,22 @@ impl<'a> Builder for EditChannel<'a> {
     ///
     /// [Manage Channels]: Permissions::MANAGE_CHANNELS
     /// [Manage Roles]: Permissions::MANAGE_ROLES
-    async fn execute(
+    #[cfg(feature = "http")]
+    pub async fn execute(
         self,
         cache_http: impl CacheHttp,
-        ctx: Self::Context<'_>,
-    ) -> Result<Self::Built> {
+        chan_id: ChannelId,
```

Comment:
I did this because it makes the below usage of it blow the line limit and expand a ton, although I guess it doesn't matter really. Would you prefer the consistent naming over caring about lines?

Reply:
Consistent naming is better. There's plenty of other places where calls to `user_has_perms_cache`end up wrapping.

Reply:
Fair enough, good to merge after fixed?